website/playground/src/DesktopPlayground.tsx

```diff
@@ -39,6 +37,10 @@ export default function DesktopPlayground({
 
 	const romeAstCodeMirrorExtension = [romeAst()];
 	const romeFormatterIrCodeMirrorExtension = [RomeFormatterIr()];
+	//
+	const romeAstSyntacticDataRef = useRef(null as RomeAstSyntacticData | null);
```

Comment:
```suggestion
	const romeAstSyntacticDataRef = useRef<RomeAstSyntacticData | null>(null);
```

Not sure if this suggestions does the same, because I have never used this kind of syntax before

Reply:
The suggestion should be applied to the other ref too.